conf/modules/fdm_jsbsim.xml

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="fdm_jsbsim" dir="fdm">
+  <doc>
+    <description>
+      JSBSim FDM backend for NPS simulator
+
+      Use the paparazzi-jsbsim package if it is installed, otherwise look for JSBsim under /opt/jsbsim
+    </description>
+    <configure name="JSBSIM_ROOT" value="/opt/jsbsim" description="set root directory for JSBSim library"/>
```

Comment:
description should make it more clear that this is only used if `JSBSIM_PKG` is `no`